main-command/src/main/scala/sbt/internal/client/NetworkClient.scala

```diff
@@ -330,7 +330,7 @@ class NetworkClient(
 
         val cmd = arguments.sbtLaunchJar match {
           case Some(lj) =>
-            List("java") ++ arguments.sbtArguments ++
+            List(s"${Properties.javaHome}/bin/java") ++ arguments.sbtArguments ++
```

Comment:
The `Properties.javaHome` is probably null when using the native client (sbtn) so it is probably safer to check for null.

```suggestion
            val java = if (Properties.javaHome != null) s"${Properties.javaHome}/bin/java" else "java"
            List(java) ++ arguments.sbtArguments ++
```

Also it is worth noting that https://github.com/sbt/sbt/pull/6593 makes this branch of the match statement less useful and we would rather prefer to use the sbt script.

Reply:
OK thanks for your feedback, maybe we can close this PR if yours is already fixing the problem.

Reply:
This is still useful when someone uses IntelliJ and does not have java nor sbt installed.